lib/ui/screen/root.dart

```diff
@@ -28,23 +29,35 @@ class RootScreen extends ConsumerStatefulWidget {
 class _RootScreenState extends ConsumerState<RootScreen> {
   ScrollController? _primaryScrollController;
 
+  bool _isTopPosition = true;
+
   @override
   Widget build(BuildContext context) {
     final localization = ref.watch(localizationProvider);
 
     final screenSize = context.screenSize;
     final currentTab = RootTab.current(context);
 
+    void scrollOffsetListener() {
+      final offset = _primaryScrollController?.offset ?? 0;
+      setState(() {
+        _isTopPosition = offset < 50.0;
+      });
```

Comment:
細かいですが、ここの`setState` はスクロールで少しでも動く度に `RootScreen` 全体でリビルドが走るので最適化しておいたほうがいいかなと思います。
```suggestion
      final isTopPosition = offset < 50.0;
      if (_isTopPosition != isTopPosition) {
        setState(() {
          _isTopPosition = offset < 50.0;
        });
      }
```

Reply:
[a4a90b9](https://github.com/FlutterKaigi/conference-app-2023/pull/249/commits/a4a90b92fd566f1d390d3e20c568d7fc82a66326)で対応しました！